core/src/Kokkos_View.hpp

```diff
@@ -1725,6 +1752,60 @@ class View : public ViewTraits<DataType, Properties...> {
                   "Layout is not constructible from extent arguments. Use "
                   "overload taking a layout object instead.");
   }
+
+  //----------------------------------------
+  // MDSpan converting constructors
+#ifdef KOKKOS_ENABLE_IMPL_MDSPAN
+  template <typename U = typename Impl::MDSpanViewTraits<traits>::mdspan_type>
+  KOKKOS_INLINE_FUNCTION KOKKOS_IMPL_CONDITIONAL_EXPLICIT(traits::is_managed)
+      View(const typename Impl::MDSpanViewTraits<traits>::mdspan_type& mds,
```

Comment:
Pass it by value maybe
Same comment below

Reply:
I'm not sure about this, for fully dynamic extents, the size of the mdspan could be up to 64 bytes.

Reply:
Withdrawn